angularjs-portal-frame/src/main/webapp/portal/main/partials/welcome-modal-template.html

```diff
@@ -0,0 +1,12 @@
+<div class="welcome-popup center">
+  <h1 ng-bind-html="welcome.title"></h1>
+  <img src="{{ welcome.img }}" class="hidden-xs">
+	<div class="sr-only" tabindex="0" id="welcomeContent" aria-labelledby="welcomeContent">
+		<p>{{ welcome.accessibilityText }}</p>
+	</div>
+  <!-- <p class="visible-xs-* learn-more">Learn more at the <a href="#">KnowledgeBase.</a></p> -->
```

Comment:
Idea: What if the display of the learn more links were predicated on the `learnMoreURL` data having a non-empty value? Such that this HTML template could ship ready to display the learn more link, and would display it if the `welcome.json` were to start including that value.  That might put this even closer to ready to go such that the KB URL could be added to just the one file if it emerges late towards the August 11th release.